.gitlab.settings.xml

```diff
@@ -0,0 +1,81 @@
+<settings>
+    <profiles>
+        <profile>
+            <id>internal</id>
+            <properties>
+                <eva.mongo.host>${env.MONGO_HOST_DEVELOPMENT}</eva.mongo.host>
```

Comment:
${env.VARIABLE_NAME} will be substituted with the Gitlab defined variables during the Gitlab CI pipeline execution.

Note the different syntax when using the variables in the .gitlab-ci.yaml file (simple $VARIABLE_NAME in that file)